docs/jwst/references_general/extract1d_reffile.inc

```diff
@@ -83,6 +83,16 @@ and background extraction regions.
 See :ref:`extract-1d-for-slits` for more details on how these parameters
 are used in the extraction process.
 
+Editing JSON Reference File Format for non-IFU data
++++++++++++++++++++++++++++++++++++++++++++++++++++
+The default EXTRACT1D reference file is found in CRDS. The user can download this file, modify the contents,
+and use this modified file in ``extract_1d`` by specifying this modified reference file with the ``override`` option
+(:ref:`override in python <override_ref_python>` or :ref:`override in strun <override_ref_strun>`). The format for JSON files has to be exact, for example,
+the format of a floating point with a fractional portion must include at least one decimal digit, so "1." is invalid, while "1.0" is valid.
+The best practice after editing a JSON reference file is to run a JSON validator off-line and correct any format errors before using
+the JSON reference file in the pipeline.
+
```

Comment:
Can we suggest any particular JSON validators and/or where to get them?  Or do we not want to promote any particular one?

Reply:
I was going to add a suggestion to use jsonlint.com to use, but I was not sure if we wanted to promote one. I asked a few people that they use and I was pointed each time to that option.
Should I add a link to that one ?

Reply:
I think it wouldn't hurt to mention it in a non-promotional way, i.e. something like "... run a JSON validator off-line, such as <jsonlint.com>, ..."